docs/node/getting-started.md

```diff
@@ -0,0 +1,61 @@
+# Getting started
```

Comment:
I want to see this converted to the introduction.md file with a title of "Node SDK introduction"